languages.toml

```
@@ -860,6 +860,7 @@ file-types = [
   "tcshrc",
   "bashrc_Apple_Terminal",
   "zshrc_Apple_Terminal",
+  "conf",
```

```suggestion
  { glob = "tmux.conf" },
```

`conf` as a file-type is already taken by another language but we can specifically match `tmux.conf` as a full filename here instead for the same effect